.idea/aws.xml

```diff
@@ -2,14 +2,15 @@
 <project version="4">
   <component name="accountSettings">
     <option name="activeProfile" value="profile:perspective-svozza" />
-    <option name="activeRegion" value="us-east-1" />
+    <option name="activeRegion" value="eu-west-1" />
     <option name="recentlyUsedProfiles">
       <list>
         <option value="profile:perspective-svozza" />
```

Comment:
Should this file be gitignored?

Reply:
This is very weird, I have that whole folder where that file lives excluded in my `.gitignore` folder.